src/java/org/apache/nutch/util/SitemapProcessor.java

```diff
@@ -402,9 +402,8 @@ public void sitemap(Path crawldb, Path hostdb, Path sitemapUrlDir, boolean stric
     try {
       boolean success = job.waitForCompletion(true);
       if (!success) {
-        String message = "SitemapProcessor_" + crawldb.toString()
-            + " job did not succeed, job status: " + job.getStatus().getState()
-            + ", reason: " + job.getStatus().getFailureInfo();
+        String message = NutchJob.getJobFailureLogMessage(
+            "SitemapProcessor_" + crawldb.toString(), job);
```

Comment:
I know it was already there: maybe simplify the job name in the message to simply "SitemapProcessor" and leave the path to the CrawlDb away?

Reply:
@sebastian-nagel Done. Thanks.